site/cypress/support/commands.js

```diff
@@ -118,7 +118,6 @@ Cypress.Commands.add('checkLogoutInAfterEach', () => {
  * @param {int} wait
  */
 Cypress.Commands.add('waitAndReloadUntil', (condition, timeout, wait = 100) => {
-    cy.wait(wait);
```

Comment:
I think this one is good to leave in, as that is the main purpose of the function

Reply:
Fixed in https://github.com/Submitty/Submitty/pull/10562/commits/96a3021a32be35b884d51bc2d2187d98513634e1